indra/newview/character/avatar_constraint.llsd

```diff
@@ -11,30 +11,30 @@
         },
     'mCollarLeft':
         {
-            'cone_angle':r9,
+            'cone_angle':r3,
             'forward_axis':
                 [r-0.021,r0.085,r0.165],
-            'max_twist':r18,
-            'min_twist':r-18,
+            'max_twist':r3,
+            'min_twist':r-3,
             'type':'TWIST_LIMITED_CONE'
         },
     'mCollarRight':
         {
-            'cone_angle':r9,
+            'cone_angle':r3,
             'forward_axis':
                 [r-0.021,r-0.085,r0.165],
-            'max_twist':r18,
-            'min_twist':r-18,
+            'max_twist':r3,
+            'min_twist':r-3,
             'type':'TWIST_LIMITED_CONE'
         },
     'mElbowLeft':
         {
             'forward_axis':
                 [r0,r1,r0],
             'max_bend':r0,
-            'max_twist':r135,
+            'max_twist':r0.5,
             'min_bend':r-157.5,
-            'min_twist':r-45,
+            'min_twist':r-0.5,
             'pivot_axis':
                 [r0,r0,r1],
             'type':'ELBOW'
```

Comment:
The SL avatar mesh is not weighted correctly to the forearm bone and the result is poor mesh animation when the forearm is twisted.  This is why I'm tightening the twist limits of the elbow joints to a very narrow window.  This means ll "twist" of the forearm needs to happen at the wrist... which also has poor mesh blend results but it looks better than when it happens at the forearm.